### DISCUSSION TITLE
When is the ios version coming out

### DISCUSSION THREAD

**Discussion by KyrieZhangi** [gitextensions/gitextensions#11861] [Q&A] [answered]
### Environment

- GitExtensions version:
- GIT version: 
- OS version: 
- .NET version: 

### Issue description

When is the ios version coming out

### Steps to reproduce

When is the ios version coming out

### Did this work in previous version of GitExtensions?

When is the ios version coming out

### Diagnostics

When is the ios version coming out

**Comment by RussKie** [comment, marked as answer]
When someone builds an iOS version.
